src/api/routes/summaries.ts

```diff
@@ -180,3 +190,48 @@ export async function findMissingCrudeSummaries(): Promise<CrudeSummaryDescripto
   }
   return ret;
 }
+
+export async function extractHandlebars(transcriptId: string): Promise<string[]> {
```

Comment:
move these functions into transcripts.ts because they are more related to transcripts than summaries.